tox.ini

```diff
@@ -57,6 +57,8 @@ extensions = .rst
 
 
 [testenv:releasenotes]
+install_command =
+    pip install {opts} {packages} --isolated
 commands =
     sphinx-build -a -E -W -d releasenotes/build/doctrees -b html releasenotes/source releasenotes/build/html
```

Comment:
Perhaps we should rather just make this tox env run `gating/generate_release_notes/run` and do without the `install_command` change?

Reply:
As discussed in slack, there is too much change involved in switching to this method, due to the env vars that would need to be passed in. Leaving as-is for now and will fix this is  in a tech-debt card